### PR TITLE
fix(failurechecker): use the logger!

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: packages/failurechecker/src/failurechecker.ts [hunk 1]
```diff
@@ -43,14 +43,14 @@ interface ConfigurationOptions {
 }
 
 export function failureChecker(app: Application) {
-  app.on(['schedule.repository' as any], async context => {
+  app.on('schedule.repository' as '*', async context => {
```
Comment (tbpg) [inline packages/failurechecker/src/failurechecker.ts:46]: This change seems unrelated to logging?

**Review by tbpg — CHANGES_REQUESTED** [review]

File: packages/failurechecker/src/failurechecker.ts (hunk 1 above)
Comment (JustinBeckwith) [reply packages/failurechecker/src/failurechecker.ts:46]: This was just a little bit of type gymnastics to prevent eslint warnings where we didn't add 'ignore no-any' comments elsewhere.  It has no material change on the underlying code.  Happy to remove if it's problematic

**Review by tbpg — APPROVED** [review]
